packages/variants/src/variants.ts

```diff
@@ -15,10 +16,32 @@ function makeThemeFromPartials(
 }
 
 /**
- * A variant where the background soft shade of the neutral color. Most other colors remain unchanged.
+ * Returns the specified variant theme for the given theme.
+ * IMPORTANT: Never generate a variant from a variant.
```

Comment:
I would use softer language. It's not recommended (results will be ugly), but certainly possible.